AUTHORS

```diff
@@ -101,4 +102,4 @@ Vladimír Chalupecký <vladimir.chalupecky@gmail.com>
 Yevgeniy Vahlis <evahlis@gmail.com>
 Yucheng Zhu <zyctc000@gmail.com>
 Zoe Juozapaitis
-Patrick DeVivo <patrick@tickgit.com>
+Patrick DeVivo <patrick@tickgit.com>
```

Comment:
Sorry. Can you repair this and in CONTRIBUTORS. It will be changed later, but just to minimised diff here.

Reply:
That was careless of me.